easybuild/toolchains/gcc.py

```diff
@@ -29,9 +29,11 @@
 """
 
 from easybuild.toolchains.gcccore import GCCcore
+from easybuild.tools.toolchain import DUMMY_TOOLCHAIN_NAME
 
 class GccToolchain(GCCcore):
     """Simple toolchain with just the GCC compilers."""
     NAME = 'GCC'
     COMPILER_MODULE_NAME = [NAME]
-    SUBTOOLCHAIN = GCCcore.NAME
+    SUBTOOLCHAIN = [GCCcore.NAME, DUMMY_TOOLCHAIN_NAME]
```

Comment:
@bartoldeman How does this relate to `--add-dummy-to-minimal-toolchains`?

`dummy` should only be considered a subtoolchain when `--add-dummy-to-minimal-toolchains` is enabled?

Reply:
Actually, it's probably OK, since `dummy` was already a subtoolchain for `GCCcore` (see below)..

Reply:
exactly. the bfs tree has dummy twice here:
```
       gcc
     /     \
gcccore    dummy
    |
  dummy
```
which actually corresponds to pre-existing behaviour since the legacy gcc toolchains do not have gcccore. `det_toolchain_version` then explicitly filters out dummy if `--add-dummy-to-minimal-toolchains` is not set.

Reply:
hmm, should adjust the ascii-art diagram comment then!

Reply:
Yes, indeed :)

It's basically this:

```
         gcc
       /     \
     /        \
gcccore       |
        \    /
        dummy
```